truco_mobile/lib/src/model/card_model.dart

```diff
@@ -0,0 +1,23 @@
+class Card {
+  final String rank;
+  final String suit;
+  final int value;
```

Comment:
Outra sugestão talvez seria legal fazer os getters and setters para faciliar a lógica das manilhas que vão alterar o valores da card ou a comparação dos valores

Reply:
Sim também teremos que criar os métodos para facilitar o uso de outras funções.